Spring classes announcement work in progress (#229)
SFPC/sfpc.github.io: recreatingthepast-spring2020.md

---
title: Recreating the Past 
layout: participate
location:
contact: sfpc
slides:

  - "/static/img/recreating/img___0000_Layer 9.jpg"
  - "/static/img/recreating/img___0001_Layer 7.jpg"
  - "/static/img/recreating/img___0002_Layer 8.jpg"
  - "/static/img/recreating/img___0003_Screen Shot 2020-04-29 at 10.08.19 AM.jpg"
  - "/static/img/recreating/img___0004_Screen Shot 2020-04-29 at 10.06.23 AM.jpg"
  - "/static/img/recreating/img___0006_Screen Shot 2020-04-29 at 10.01.58 AM.jpg"
  - "/static/img/recreating/img___0007_Screen Shot 2020-04-29 at 10.00.54 AM.jpg"
  - "/static/img/recreating/img___0008_Layer 6.jpg"
  - "/static/img/recreating/img___0009_Layer 4.jpg"
  - "/static/img/recreating/img___0010_Screen Shot 2020-04-28 at 11.52.13 AM.jpg"
  - "/static/img/recreating/img___0011_Screen Shot 2020-04-28 at 11.51.36 AM.jpg"
  - "/static/img/recreating/img___0012_Screen Shot 2020-04-28 at 11.51.46 AM.jpg"
  - "/static/img/recreating/muriel.jpg"
 
---

### Recreating the Past 

> - Date: Starting in May 21st, 2020, once a week on Thursday 1-3pm EST for ten weeks 

Reviewer: ```suggestion
- Date: Starting on May 21st, 2020. Once a week on Thursdays from 1-3pm EST, for ten weeks.
```

- Teacher: Zach Lieberman 
- [Application](https://airtable.com/shrTfxvRtZ8nsUTyu) is now open until May 12th, 2020

### Class Description

> In Recreating the Past, we will study computational art from the past decades and recreate these works with contemporary techniques to gain aesthetic, analytical and technical knowledge. The class is led by artist and SFPC co-founder Zach Lieberman, and will be offered online for the first time. For each class, we investigate a different set of artists that have created work with software (either through code or algorithmic processes) and then recreate their artworks from scratch, discussing their approaches and tools, as well as poetic underpinnings. By recreating historic works, we also investigate what modern tools and approaches have to offer. In addition, we will also look at broader examples of what the artwork teaches us about algorithmic approaches and conceptual explorations. For example, seminal works like Camille Utterback and Romy Achituv's "Text Rain" can be used as a starting point for discussing computer vision-based interaction. The class will focus on the building blocks of media art and generative design practice, such as algorithmic composition, typography, imaging, computer vision, audio analysis / synthesis, and interaction design. 

Reviewer: ```suggestion
In Recreating the Past, we will study computational art from decades past, and recreate these works with contemporary techniques to gain aesthetic, analytical and technical knowledge. The class is led by artist and SFPC co-founder Zach Lieberman, and will be offered online for the first time. For each class, we investigate a different set of artists that have created work with software (either through code or algorithmic processes) and then recreate their artworks from scratch, discussing their approaches and tools, as well as poetic underpinnings. By recreating historic works, we also investigate what modern tools and approaches have to offer. In addition, we will also look at broader examples of what the artwork teaches us about algorithmic approaches and conceptual explorations. For example, seminal works like Camille Utterback and Romy Achituv's "Text Rain" can be used as a starting point for discussing computer vision-based interaction. The class will focus on the building blocks of media art and generative design practice, such as algorithmic composition, typography, imaging, computer vision, audio analysis / synthesis, and interaction design. 
```

> Students will be encouraged to begin a practice of generative visual sketching in whatever medium they feel comfortable in and share their process amongst the cohort. Class examples will be prepared in openFrameworks which is written in C++ and built on top of OpenGL. Although the assignments will be required to be completed via coding, the class is open to a variety of skill and experience levels. The lead teacher and teaching assistant will help to get you up to speed. Students are expected to have basic knowledge of coding and familiarity with using variables, functions, control structures. If you are a beginner, we recommend you take SFPC Coding Bootcamp before applying for this class. 

Reviewer: ```suggestion
Students will be encouraged to begin a practice of generative visual sketching in whatever medium they feel comfortable in and share their process amongst the cohort. Class examples will be prepared in openFrameworks which is written in C++ and built on top of OpenGL. Although the assignments will be required to be completed via coding, the class is open to a variety of skill and experience levels. The lead teacher and teaching assistant will help to get you up to speed. Students are expected to have basic knowledge of coding and familiarity with using variables, functions, and control structures. If you are a beginner, we recommend you take SFPC Coding Bootcamp before applying for this class. 
```

### Syllabus 

Here are examples of some of the artists / works / ideas we will be exploring:

- Chaos and Order (Vera Molnar)
- Animation and Harmony (John Whitney)
- Computational Typography (Muriel Cooper / John Maeda)
- Pattern (Bridgette Riley / Anni Albers)
- Pixels (Ken Knowlton / Lillian F. Schwartz)
- Images (Nancy Burson / Jason Salavon) 
- Body (Camille Utterback and Romy Achituv / Myron Krueger)

> - Glitch / Deformation  (Rosa Menkman / Steina and Woody Vasulka)

Reviewer: ```suggestion
- Glitch / Deformation (Rosa Menkman / Steina and Woody Vasulka)
```

![](/static/img/recreating/recreatingthepast2.png) 

### From our students

"now, being months after taking this course, I think back on RTP often because the works we recreated are *so* relevant to the designs we see in our everyday walk down the street or strut through an exhibit. I look at fonts and typography and placement on posters. I analyze digital aesthetic structures with a more informed eye because I learned the how/what/why of appreciating the past. RTP gave me technical tools to learn how to appreciate the bedrock of the future - our past." Oceane Boulais (MIT) 

"RTP is a great class with a strong workshop vibe. Using past works as a basis for creating new pieces is liberating, and it's gratifying to see all the pieces come together as an output of this journey! Zach's teaching style is refreshing as it blends discussions about different artistic perspectives with some practical coding tips to help us materialize our ideas." Joanne Leong (MIT)

> "Yes, Recreating The Past taught me a lot about coding. But for me, it’s value as an art history class, as an exercise in the importance of archiving, preservation, and curation, felt like a home I’d always been searching for. History is not a passive subject you simply consume, but an active form, something you “do,” and this was one of the only classes I’ve ever been in that seemed to not only understand that, but revel in it.” Max Neely Cohen (SFPC)

Reviewer: ```suggestion
"Yes, Recreating The Past taught me a lot about coding. But for me, its value as an art history class, as an exercise in the importance of archiving, preservation, and curation, felt like a home I’d always been searching for. History is not a passive subject you simply consume, but an active form, something you “do,” and this was one of the only classes I’ve ever been in that seemed to not only understand that, but revel in it.” Max Neely Cohen (SFPC)
```

> "I loved it! It was the most balanced class in the course, with enough theory and history to make the practice more enjoyable.I particularly like the title of the class. It’s a way of saying something else. It could have been called “code”, but the name talks a lot about other things." Javier De Azkue (SFPC)

Reviewer: ```suggestion
"I loved it! It was the most balanced class in the course, with enough theory and history to make the practice more enjoyable. I particularly like the title of the class. It’s a way of saying something else. It could have been called “code”, but the name talks a lot about other things." Javier De Azkue (SFPC)
```

> “In Recreating the Past, I learned to look closely at works of art in a way that I had never done before. It was an engaging introduction to the history of computational art that helped orient my search for more inspiration in the space. Most importantly, RTP reminded me of the creative possibility of mathematical and algorithmic thinking in the context of art.” Bomani Oseni McClendon  (SFPC)

Reviewer: ```suggestion
“In Recreating the Past, I learned to look closely at works of art in a way that I had never done before. It was an engaging introduction to the history of computational art that helped orient my search for more inspiration in the space. Most importantly, RTP reminded me of the creative possibility of mathematical and algorithmic thinking in the context of art.” Bomani Oseni McClendon (SFPC)
```

### How do I know if this class is right for me?

This class is right for you if: 

> - You have a basic knowledge of code (for example, you’ve taken SFPC bootcamp class or have experience with coding in p5, etc)

Reviewer: ```suggestion
- You have a basic knowledge of code (for example, you’ve taken SFPC's bootcamp class or have experience with coding in p5, etc.)
```

> - You have access to a computer that can run openFrameworks.

Reviewer: ```suggestion
- You have access to a computer that can run [openFrameworks](https://openframeworks.cc).
```

- You are interested in learning the craft of creative coding 
- You enjoy researching / learning about how artists work

> - You agree with SFPC Code of Conduct  

Reviewer: ```suggestion
- You agree with SFPC's [code of conduct](https://github.com/SFPC/codeofconduct/blob/master/README.md) 
```
@tchoi8 should we link to the code of conduct here?

- You enjoy working with others 

This class is not right for you if: 

-  You have no experience with code

> -  You are interested in a more skills based, less theoretical approach to learning creative code

Reviewer: ```suggestion
-  You are interested in a more skills-based, less theoretical approach to learning creative code
```

-  You are not interested in research or engaging with artworks from the past

![](/static/img/recreating/recreatingthepast3.png) 

### How will this class work online?

> The class will be taught for 2 hours weekly with an additional ½ hour to 1 hr or asynchronous content which includes short lectures and code explanations. The homework will take additional time per week depending on the complexity of the artwork being recreated (Estimated 1-4 hours / week). In addition the teacher and TA will each offer 2 hours of office hour support, at a scheduled time TBD throughout the week.  We will use github, dropbox paper, and slack as our primary tools with zoom used for the class meetings and TA hours. Recordings of the live portions of each class will be archived and shared with participants for a limited time. 

Reviewer: ```suggestion
The class will be taught for 2 hours weekly with an additional ½ hour to 1 hr or asynchronous content which includes short lectures and code explanations. The homework will take additional time per week depending on the complexity of the artwork being recreated (estimated 1-4 hours / week). In addition the teacher and TA will each offer 2 hours of office hour support, at a scheduled time TBD throughout the week.  We will use GitHub, Dropbox Paper, and Slack as our primary tools, with Zoom used for the class meetings and TA hours. Recordings of the live portions of each class will be archived and shared with participants for a limited time. 
```

![](/static/img/recreating/zach-lieberman.jpg) 

### Instructors
 

> Teacher: [Zachary Lieberman](https://instagram.com/zach.lieberman) is an artist, researcher, and educator with a simple goal: he wants you surprised. In his work, he creates performances and installations that take human gestures as input and amplify them in different ways -- making drawings come to life, imagining what the voice might look like if we could see it, transforming people's silhouettes into music.  He's been listed as one of Fast Company's Most Creative People and his projects have won the Golden Nica from Ars Electronica, Interactive Design of the Year from Design Museum London as well as listed in Time Magazine's Best Inventions of the Year.  He creates artwork through writing software and is a co-creator of openFrameworks, an open source C++ toolkit for creative coding and helped co-found the School for Poetic Computation, a school examining the lyrical possibilities of code.  He is also an associate adjunct professor at MIT’s Media Lab where he runs the Future Sketches group. 

Reviewer: ```suggestion
Teacher: [Zachary Lieberman](https://instagram.com/zach.lieberman) is an artist, researcher, and educator with a simple goal: he wants you surprised. In his work, he creates performances and installations that take human gestures as input and amplify them in different ways — making drawings come to life, imagining what the voice might look like if we could see it, transforming people's silhouettes into music.  He's been listed as one of Fast Company's Most Creative People and his projects have won the Golden Nica from Ars Electronica, Interactive Design of the Year from the Design Museum in London, and have been listed in Time Magazine's Best Inventions of the Year.  He creates artwork through writing software and is a co-creator of openFrameworks, an open source C++ toolkit for creative coding. He co-founded the School for Poetic Computation, a school examining the lyrical possibilities of code, and is also an associate adjunct professor at MIT’s Media Lab where he runs the Future Sketches group. 
```

### How do I apply?

[Application](https://airtable.com/shrTfxvRtZ8nsUTyu) is now open until May 12th, 2020. We evaluate applicants based on their written responses and work sample. We prioritize diversity of our students. In some cases, we may recommend you to participate in another way, if you are too experienced for the class or if you need to take pre-requisite to fully take advange of the class.    

We will be accepting up to 15 students on a rolling basis. Rolling admissions means there are fewer and fewer slots the longer you wait, so if you’re interested in the program get your application in early! If you are accepted, you will get a link to Eventbrite where you can register and make payment.  
 
### How much is tuition?

> $1200 USD + Eventbrite Processing Fees. Upon payment, your space in the class will be reserved.

Reviewer: ```suggestion
$1200 USD + Eventbrite processing fees. Upon payment, your space in the class will be reserved.
```

> We are completely self-funded, which dramatically limits our ability to offer scholarships. While we price our classes reasonably to offer quality learning experiences, we understand the tuition is beyond the reach of many people. We are sensitive to the rapidly changing economy due to the COVID-19. While we can't offer scholarships like the past, we hope to support students in need. Our previous students have received support from cultural foundations, schools, current employers, or crowdfunding. We are happy to provide supporting materials as proof of acceptance. Please check out these resources for financial support in response to the COVID-19 pandemic.

Reviewer: ```suggestion
We are completely self-funded, which dramatically limits our ability to offer scholarships. While we price our classes reasonably to offer quality learning experiences, we understand the tuition is beyond the reach of many people. We are sensitive to the rapidly changing economy due to covid-19. While we can't offer scholarships like the past, we hope to support students in need. Our previous students have received support from cultural foundations, schools, current employers, or crowdfunding. We are happy to provide supporting materials as proof of acceptance. Please check out these resources for financial support in response to the covid-19 pandemic.
```

- [Cloud9 Mutual Aid Doc: Financial Assistance](https://docs.google.com/document/d/1Qo_w8b6u2yXKzE7dIUmSeWqk3FFrqS1KhoCGzqcmZiQ/edit#heading=h.8jojokwzkoa7)
- [The Art of Social Distancing](https://docs.google.com/spreadsheets/d/e/2PACX-1vTt0lJMLDRlx_HsE132C3aGFa-D_rvk8rDVtkt9E7BH0jVQHrv-zD0favR98AtgTlPbNl2A5RPDH63X/pubhtml)

> We are committed to being fully transparent about how we make and spend money. In the spirit of radical openness and generosity, on which the school was founded, [we open-source our finances on Github](https://github.com/sfpc/finance-and-administration). There, you can read financial reports and download raw statements.

Reviewer: ```suggestion
We are committed to being fully transparent about how we make and spend money. In the spirit of radical openness and generosity, on which the school was founded, [we open-source our finances on GitHub](https://github.com/sfpc/finance-and-administration). There, you can read financial reports and download raw statements.
```

### What if I can’t go, can I get a refund?
- Yes, we can give you 100% refund up to 10 days before class starts
- 50% refund after 10 days, until the first day of class
- No refunds can be given after the first day of class

### Accessibility 

> Our classes are conducted in spoken English with audiovisual materials such as slides, code examples and video. All classes will be held over Zoom until further notice. Please contact us if you need a special accessibility requests. In the past, we worked with American Sign Language Interpreters and Visual Interpreters, as well as accessibility and disability rights organizations to make our classes more inclusive. Our programs are rapidly changing due to the COVID-19, we are adjusting to the needs of our community. Please help us become more accessible and inclusive. info@sfpc.io 

Reviewer: ```suggestion
Our classes are conducted in spoken English with audiovisual materials such as slides, code examples and video. All classes will be held over Zoom until further notice. Please contact us if you need a special accessibility requests. In the past, we worked with American Sign Language Interpreters and Visual Interpreters, as well as accessibility and disability rights organizations to make our classes more inclusive. Our programs are rapidly changing due to covid-19 and we are adjusting to the needs of our community. Please help us become more accessible and inclusive. info@sfpc.io 
```
